test/export/ValueSetExporter.test.ts

```diff
@@ -249,6 +249,21 @@ describe('ValueSetExporter', () => {
     expect(loggerSpy.getLastMessage('warn')).toMatch(warning);
     expect(loggerSpy.getLastMessage('warn')).toMatch(/File: Wrong\.fsh.*Line: 2 - 5\D*/s);
   });
+  it('should export a value set with an extension', () => {
+    const valueSet = new FshValueSet('BreakfastVS');
+    valueSet.title = 'Breakfast Values';
+    const extensionRule = new CaretValueRule('');
+    extensionRule.caretPath = 'extension[structuredefinition-fmm].valueInteger';
+    extensionRule.value = 1;
+    valueSet.rules.push(extensionRule);
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export().valueSets;
+    expect(exported.length).toBe(1);
+    expect(exported[0].extension).toContainEqual({
+      url: 'http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm',
+      valueInteger: 1
+    });
+  });
```

Comment:
Similarly to the tests for `CodeSystemExporter`, there's a section for caret value rule tests around line 1580.

Reply:
Thank you! Moved the test in commit [b0d7c38](https://github.com/FHIR/sushi/pull/1074/commits/b0d7c38dcc19e422d2a6cf9915ac96ebfb6a9890).